browser-only.js

```diff
@@ -28,6 +28,9 @@ if (typeof window !== 'undefined') {
   // Polyfill window.matchMedia (primarily for IE9)
   require('matchmedia-polyfill');
   require('matchmedia-polyfill/matchMedia.addListener');
+
+  // Polyfill CustomEvent
+  require('custom-event-polyfill');
```

Comment:
Any reason to prefer this one, vs https://www.npmjs.com/package/custom-event? I have no preference, just want to make sure we're using the best/most maintained one.